.github/workflows/operator_benchmark.yml

```diff
@@ -28,9 +28,9 @@ permissions:
   contents: read
 
 jobs:
-  opbenchmark-build:
+  x86-opbenchmark-build:
```

Comment:
Nit: I wonder if a strategy would work here, but having separate build and test jobs for x86 and aarch64 works too, and probably simpler

Reply:
Sure, I can do the matrix in followup PRs